DigitalLearningSolutions.Web/Scripts/trackingSystem/reports.ts

```diff
@@ -71,24 +75,37 @@ request.open('GET', path, true);
 request.responseType = 'json';
 request.send();
 
-let allRowsShown = false;
-const viewMoreLink = <HTMLLIElement>document.getElementsByClassName('js-shown-load-more').item(0);
-viewMoreLink.style.display = 'block';
-
 viewMoreLink.addEventListener('click', (event) => {

```

Comment:
The page-load code that sets up the toggle-activity-rows button could be extracted into a function called `setUpToggleActivityRowsButton()` or such like.